wiki/Play_Styles/en.md

```diff
@@ -1,165 +1,169 @@
 
 ### Joystick
 
-A possible play-style for players used to pixel-old arcade cabinets' control scheme.
+Using a joystick to control the catcher is a play style for players who are used to pixel-old arcade cabinets' control scheme.
 
-Players move the head of the joystick and press the face button/trigger/keyboard key to trigger the _Dash!_ effect.
+Players move the head of the joystick and press the face button/trigger/keyboard key to trigger the *Dash!* effect.
 
 ### Gamepad
 
-A possible play-style for players comfortable playing with a gamepad.
+Using a gamepad to control the catcher is a play style for players who prefer the feeling of a particular D-Pad or control stick as opposed to a keyboard.
+
+To move the catcher left and right, some players prefer to press the left/right arrows on their gamepad's D-Pad to move, while others prefer to use control stick directions. Though, due to a gamepad's wide variety of buttons, players can use triggers, bumpers, or other face buttons to move the  catcher.
 
-Players press the left/right arrow of the gamepad to move the catcher left/right and press then hold the face button to trigger the _Dash!_ effect.
+To trigger the *Dash!* effect, many players use one of the face buttons on their gamepad. Though, some players prefer to use triggers, bumpers, or control sticks. 
 
 ## ![](/wiki/shared/mode/mania.png) osu!mania
 
 ### Keyboard
 
 [External guide for Mechanical Keyboard](http://www.daskeyboard.com/blog/mechanical-keyboard-guide/)
 
-Keyboard was once the _only_ way to play osu!mania until other playing methods was supported over time.
-
-Players place their fingers on the keyboard keys with respect to the key-bindings and currently playing Keys setting.
-The advantage of this play-style is that it can accommodate basically all the Keys setting.
+Keyboard was once the *only* way to play osu!mania until other play methods was supported over time.
 
-The only catch-point is getting used to it and generic keyboard's ghosting effect/membrane keys.
+To play with a keyboard, players would press down the keys respective to their placement on the in-game keyboard. By default these keys are DFJK. The advantage of this play-style is that it can accommodate basically all the Keys setting.
 
 ### DJMAX/O2Jam/Beatmania IIDX/etc. arcade controller
 
-The play-style tuned for seasoned veterans at conveyor-type rhythm games.
+These types of arcade controllers are a play-style tuned for seasoned veterans at conveyor-type rhythm games.
 
-Since 8 Keys requires mapper's decision or Game Modifiers, most players owning these arcade controllers can handle up to 7Keys (max default) and 1 special.
+*Note: Since the use of 8 Keys requires the mapper's decision or the use of Game Modifiers, most players owning these arcade controllers can handle up to 7 Keys (max default) and 1 special.*
```

Comment:
```suggestion
*Note: Since the use of 8 Keys requires the mapper's decision or the use of game modifiers, most players owning these arcade controllers can handle up to 7 keys (max default) and 1 special.*
```